meta/locations/lambertus-office.md

```diff
@@ -18,11 +18,12 @@ building is open).
 
 These individuals have been provisioned access to the office:
 
-- [Matthew](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/hewillyeah.md)
-- [@stopwatchtt](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/stopwatchtt.md)
 - [@imthesquid](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/imthesquid.md)
 - [Ray](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/rayhanadev.md)
 - [@phineas1500](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/phineas1500.md)
 - [@firefoxofficial](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/firefoxofficial.md)
+- [@infinidoge](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/infinidoge.md)
+- [@jaddenki](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/jaddenki.md)
+- Three others not in Evergreen yet
```

Comment:
Can you just list their GitHub IDs here? You don't have to link them. We can link them later once they are added to dark-forest